ui/src/config.js

```diff
@@ -43,32 +43,40 @@ export const appConfig = {
   scaling: {
     maxAllowedReplica: process.env.REACT_APP_MAX_ALLOWED_REPLICA
       ? parseInt(process.env.REACT_APP_MAX_ALLOWED_REPLICA)
-      : 10
+      : 10,
   },
   pagination: {
-    defaultPageSize: 10
+    defaultPageSize: 10,
   },
   tables: {
     defaultTextSize: "s",
     defaultIconSize: "s",
-    dateFormat: "YYYY-MM-DDTHH:mm.SSZ"
-  }
+    dateFormat: "YYYY-MM-DDTHH:mm.SSZ",
+  },
+  podLogs: {
+    // Interval (in ms) between API calls to Logs API
+    pollInterval: 10000,
```

Comment:
Is 10s too long? I believe this was 5s previously.

Reply:
`pollInterval` is only used when no new log entries are received in the current request. Otherwise, a new request with updated `since_time` query parameter is fired immediately. So this has very little effect on the pods (routers/jobs) with frequently updated logs data, but reduces the number of API calls if there is no updates (i.e. router is scaled down to 0 replicas)